src/bundle/Resources/public/js/scripts/core/custom.tooltip.js

```diff
@@ -0,0 +1,22 @@
+(function(doc) {
+    jQuery(document).ready(() => {
+        const tooltipsNode = doc.querySelectorAll('[data-toggle="tooltip"]');
+        let delay, extraClass;
+
+        for (tooltipNode of tooltipsNode) {
+            delay = {
+                show: tooltipNode.getAttribute('data-delay-show') || 150,
+                hide: tooltipNode.getAttribute('data-delay-hide') || 75,
+            };
+            extraClass = tooltipNode.getAttribute('data-extra-class') || '';
```

Comment:
I would name it `extra-classes` to be consistent with other places and this may contain multiple classes